admin/makedist.sh

```diff
@@ -45,7 +45,7 @@ for i in $*; do
 				DOMD5SUM="yes"
 			;;
 		--*)
-			echo "ERROR: Unrecognized option $i"
+			echo "ERROR! Unrecognized option $i"
```

Comment:
> Why?
> "error: foo" is a common pattern

its either ```Error: unrecognized option $i``` lowercase u -  or as I put it.



> And this is just an internal script, I don't understand why would you change things just for the sake of it... again.

Does matter. Your opinion is OK to leave everything in an inconsistent state, just because you dont care.